apiserver/facades/client/application/deployrepository.go

```diff
@@ -182,7 +190,9 @@ func (api *DeployFromRepositoryAPI) DeployFromRepository(ctx context.Context, ar
 				Status: status.Unset,
 				Since:  ptr(api.clock.Now()),
 			},
-		}, unitArgs...)
+		},
+		api.relationService.CreatePeerRelations,
```

Comment:
We don't allow service calls from other service calls. This can be dangerous and we prohibited after secret domain started doing it and we had to unwind all of it.